client-core/src/wallet/default_wallet_client.rs

```diff
@@ -105,161 +104,148 @@ where
     fn new_wallet(
         &self,
         name: &str,
-        passphrase: &SecUtf8,
+        enckey: &SecKey,
         wallet_kind: WalletKind,
     ) -> Result<Option<Mnemonic>> {
         #[cfg(not(debug_assertions))]
-        check_passphrase_strength(name, passphrase)?;
+        check_seckey_strength(name, enckey)?;
```

Comment:
I don't think there is any point in checking strength of `SecKey`. We should check strength of `passphrase` when creating/restoring a wallet.

Reply:
Done